web/packages/teleport/src/services/auth/auth.ts

```diff
@@ -40,6 +44,19 @@ const auth = {
       )
     );
   },
+  checkMfaRequired(
+    params: IsMfaRequiredRequest
+  ): Promise<{ required: boolean }> {
+    return api.post(cfg.getMfaRequiredUrl(), params);
+  },
+
+  async getAssertionResponseIfRequired(params: IsMfaRequiredRequest) {
+    const isMfaRequired = await auth.checkMfaRequired(params);
+    if (isMfaRequired.required === true) {
+      return auth.getWebauthnResponse();
+    }
+    return;
```

Comment:
Nit:

```suggestion
    if (isMfaRequired.required) {
      return auth.getWebauthnResponse();
    }
```

The second return is not necessary, is it?